### PR TITLE
Clean up FrankelementFilters

### PR DESCRIPTION
**Pull request by mhdirkse**

### CODE REVIEW

**Comment by mhdirkse** [comment]
At this point, the Frank!Doc still produces the following errors:

```
11:25:53,852 ERROR ElementRole.initConflicts():109[] Cannot resolve XML name conflict for non-deprecated FrankElement-s [nl.nn.adapterframework.extensions.sap.SapSender, nl.nn.adapterframework.extensions.sap.jco3.SapSender]
11:25:53,852 ERROR ElementRole.initConflicts():109[] Cannot resolve XML name conflict for non-deprecated FrankElement-s [nl.nn.adapterframework.extensions.sap.IdocSender, nl.nn.adapterframework.extensions.sap.jco3.IdocSender]
11:25:53,852 ERROR ElementRole.initConflicts():109[] Cannot resolve XML name conflict for non-deprecated FrankElement-s [nl.nn.adapterframework.extensions.esb.WsdlGeneratorPipe, nl.nn.adapterframework.pipes.WsdlGeneratorPipe]
11:25:53,853 ERROR ElementRole.initConflicts():109[] Cannot resolve XML name conflict for non-deprecated FrankElement-s [nl.nn.adapterframework.extensions.sap.SapListener, nl.nn.adapterframework.extensions.sap.jco3.SapListener]
11:25:53,853 ERROR ElementRole.initConflicts():109[] Cannot resolve XML name conflict for non-deprecated FrankElement-s [nl.nn.adapterframework.extensions.sap.IdocSender, nl.nn.adapterframework.extensions.sap.jco3.IdocSender]
11:25:53,853 ERROR ElementRole.initConflicts():109[] Cannot resolve XML name conflict for non-deprecated FrankElement-s [nl.nn.adapterframework.extensions.sap.SapSender, nl.nn.adapterframework.extensions.sap.jco3.SapSender]
```

File: src/main/java/org/frankframework/frankdoc/model/FrankElementFilters.java [hunk 1]
```diff
@@ -36,21 +36,6 @@ private FrankElementFilters() {
...
-		excludeFilters.add("nl.nn.adapterframework.extensions.sap.jco2.SapListener");
-		excludeFilters.add("nl.nn.adapterframework.extensions.sap.jco2.SapLUWManager");
-		excludeFilters.add("nl.nn.adapterframework.extensions.sap.jco2.IdocSender");
-		excludeFilters.add("nl.nn.adapterframework.extensions.sap.jco3.SapSender");
-		excludeFilters.add("nl.nn.adapterframework.extensions.sap.jco3.SapListener");
-		excludeFilters.add("nl.nn.adapterframework.extensions.sap.jco3.SapLUWManager");
-		excludeFilters.add("nl.nn.adapterframework.extensions.sap.jco3.IdocSender");
 		// Here are replacements for:
 		// excludeFilters.add(".*\\.IbisstoreSummaryQuerySender");
 		excludeFilters.add("nl.nn.adapterframework.webcontrol.action.IbisstoreSummaryQuerySender");
```
Comment (gvanbrakel) [inline src/main/java/org/frankframework/frankdoc/model/FrankElementFilters.java:41]: Please also remove the lines for IbisstoreSummaryQuerySender

**Review by gvanbrakel — CHANGES_REQUESTED** [review]

**Comment by mhdirkse** [comment]
After following the review comments, the following errors remain.

```
09:41:35,166 ERROR ElementRole.initConflicts():109[] Cannot resolve XML name conflict for non-deprecated FrankElement-s [nl.nn.adapterframework.extensions.sap.SapSender, nl.nn.adapterframework.extensions.sap.jco3.SapSender]
09:41:35,166 ERROR ElementRole.initConflicts():109[] Cannot resolve XML name conflict for non-deprecated FrankElement-s [nl.nn.adapterframework.extensions.sap.IdocSender, nl.nn.adapterframework.extensions.sap.jco3.IdocSender]
09:41:35,167 ERROR ElementRole.initConflicts():109[] Cannot resolve XML name conflict for non-deprecated FrankElement-s [nl.nn.adapterframework.extensions.sap.SapListener, nl.nn.adapterframework.extensions.sap.jco3.SapListener]
09:41:35,167 ERROR ElementRole.initConflicts():109[] Cannot resolve XML name conflict for non-deprecated FrankElement-s [nl.nn.adapterframework.extensions.sap.IdocSender, nl.nn.adapterframework.extensions.sap.jco3.IdocSender]
09:41:35,167 ERROR ElementRole.initConflicts():109[] Cannot resolve XML name conflict for non-deprecated FrankElement-s [nl.nn.adapterframework.extensions.sap.SapSender, nl.nn.adapterframework.extensions.sap.jco3.SapSender]
```

File: src/main/java/org/frankframework/frankdoc/model/FrankElementFilters.java [hunk 2]
```diff
@@ -36,21 +36,6 @@ private FrankElementFilters() {
 	 */
 	public static Set<String> getExcludeFilter() {
 		Set<String> excludeFilters = new TreeSet<>();
-		// Exclude classes that will give conflicts with existing, non-compatible bean definition of same name and class
-		excludeFilters.add("nl.nn.adapterframework.extensions.esb.WsdlGeneratorPipe");
```
[the remark this answers is not in the document]
Comment (mhdirkse) [reply src/main/java/org/frankframework/frankdoc/model/FrankElementFilters.java:40]: Done

File: src/main/java/org/frankframework/frankdoc/model/FrankElementFilters.java (hunk 1 above)
Comment (mhdirkse) [reply src/main/java/org/frankframework/frankdoc/model/FrankElementFilters.java]: Done.

**Comment by gvanbrakel** [comment]
Kan je dit PR nog een keer runnen? De SAP classes zijn aangepast, waardoor de errors verdwenen zouden moeten zijn

**Comment by mhdirkse** [comment]
I ran this version of the Frank!Doc on IAF SHA 059dd703c97b3ee61b22ec56b910da28dcc498ca. The Frank!Doc did not produce errors anymore. Therefore I made this PR ready-for-review.

**Review by gvanbrakel — APPROVED** [review]

**Pull request merged**